services/create_project.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import argparse
+import sys
+import click
+from typing import AnyStr
+
+sys.path.append("../")
+from models import *
+
+
+def create_client(name: AnyStr) -> None:
+    proj_response = requests.post(
+        f"{V1_API_URL}/workspaces/{WORKSPACE_ID}/clients",
+        json={"name": name},
+        headers=HEADERS,
+    )
+    print(f"Client {name} created.")
+    pass
+
+
+@click.command()
+@click.option("--name", prompt="Project name", help="The name for the project.")
+# TODO: implement clients in our DB
+def create_project(name: AnyStr) -> None:
+    colors = {
+        "e": "#69b39e",
+        "t": "#09215d",
+        "n": "#FF9800",
+        "w": "#1b60ce",
+        "c": "#8BC34A",
+    }
+
+    project_name = name.upper()
+    default_activities = Project.find(0).activities
+    proj_data = {
+        "name": project_name,
+        "isPublic": "true",  # On Clockify this means the project is visible to the whole team
+        "billable": "true",
+        "color": colors[project_name.lower()[0]],
```

Comment:
I think it's missing an exception for when the project doesn't start with does letters.